gnovm/pkg/gnolang/store.go

```diff
@@ -19,6 +19,9 @@ type PackageGetter func(pkgPath string) (*PackageNode, *PackageValue)
 // inject natives into a new or loaded package (value and node)
 type PackageInjector func(store Store, pn *PackageNode)
 
+// NativeStore is a function which can retrieve native bodies of native functions.
+type NativeStore func(pkgName string, name Name) func(m *Machine)
+
 type Store interface {
```

Comment:
Yes!

Subsequent PRs after this will remove two methods for pkg injections at least. But I do think we should probably:

- Move the defaultStore to be a concrete type used in many other places instead
- Where we do want to accept different implementation, use smaller interfaces for only the methods we need in that fn.